client/templates/components/get_certificate.html

```diff
@@ -0,0 +1,5 @@
+<h3>תג זה הינו הכרחי עבור ביצוע ההתנדבות - שמור את התג בטלפון שלך</h3>
+<img src="{{ certificate.image.url }}" alt="volunteer tag" width="400" />
+<br />
+<a href="{{ certificate.image_download_url }}">הורדת תג</a>
```

Comment:
why not?:
```suggestion
<a href="{{ certificate.image.url }}" download>הורדת תג</a>
```

Reply:
I explained this above in my comment on image_download_url function